hypervisor/arch/x86/cpu.c

```diff
@@ -430,6 +430,18 @@ void bsp_boot_init(void)
 	ASSERT(offsetof(struct run_context, ia32_spec_ctrl) ==
 		CPU_CONTEXT_OFFSET_IA32_SPEC_CTRL,
 		"run_context ia32_spec_ctrl offset not match");
+	ASSERT(offsetof(struct run_context, rflags) ==
```

Comment:
Per discussion, we want to move those OFFSET check to a separate file/API which is not needed for link...